forc-pkg/src/pkg.rs

```diff
@@ -181,8 +181,8 @@ pub enum SourceGitPinnedParseError {
 pub type DependencyName = String;
 
 pub struct PkgDiff {
-    pub added: Vec<Pkg>,
-    pub removed: Vec<Pkg>,
+    pub added: Vec<(String, Pkg)>,
+    pub removed: Vec<(String, Pkg)>,
```

Comment:
```suggestion
    pub removed: Vec<(DependencyName, Pkg)>,
```
^ Do you mind making these changes to the `added` field, as well as the `remove_deps` and `add_deps` fn signatures? `DependencyName` is just an alias for `String`, but I think it'll be useful for readability.

Reply:
Hi @kayagokalp are you able to make the change @mitchmindtree suggested above? Happy to approve this once that's been made.

Reply:
Oh, I guess I missed this one, sorry :typing